docs/source/cheatcodes.rst

```diff
@@ -0,0 +1,26 @@
+VM Cheatcodes within Titanoboa
```

Comment:
i think we should call these "custom precompiles", and clarify that they are just analogous to cheatcodes in dapptools and foundry.